src/binary/runBinary.ts

```diff
@@ -13,14 +14,19 @@ export default async function runBinary(
 
   const context = getTabnineExtensionContext();
   const args: string[] = [
-    "--client=vscode",
+    ONPREM ? "--client=vscode_on_prem" : "--client=vscode",
     "--no-lsp=true",
     tabnineExtensionProperties.logFilePath
       ? `--log-file-path=${tabnineExtensionProperties.logFilePath}`
       : null,
+    ONPREM ? "--onprem=true" : null,
+    ONPREM ? "--no_bootstrap" : null,
```

Comment:
i think it is irelevant

Reply:
we are still passing the flags. if it gets changed I will change here